documentation/specs/static-graph.md

```diff
@@ -19,12 +26,81 @@
     - [Detours](#detours)
     - [Isolation requirement](#isolation-requirement)
     - [Tool servers](#tool-servers)
+  - [Examples](#examples)
 
-# Static Graph
+## What is static graph for?
```

Comment:
Is there any visual recreation of the graph available today (like a dgml diagram or something)?  Might be an interesting future project as we invest more here.

Reply:
Interesting, we'd probably need to invest on making that more navigable and readable if we were to expose that.  Not sure if there are changes customers can do based on the graph that are worth doing.

Reply:
Yeah, if this becomes the default case I think we'll definitely need to figure out a nice visualization for it. We could do that today, and folks have done so in the past (https://www.visualstudiogeeks.com/blog/msbuild/visual%20studio/msbuild-dependency-visualizer for example), but it hasn't seemed super important.

Reply:
The binlog viewer also prints out the project dependency graph :D. But it gets pretty hairy beyond 5-10 projects. However, it's super useful to understand the p2p protocol :)